ui/app/pages/connected-sites/connected-sites.component.js

```diff
@@ -42,73 +45,129 @@ export default class ConnectSites extends Component {
     })
   }
 
-  clearSitePendingDisconnect = () => {
+  clearPendingDisconnect = () => {
     this.setState({
       sitePendingDisconnect: null,
     })
   }
 
-  disconnect = () => {
+  disconnectAccount = () => {
     const { disconnectAccount } = this.props
     const { sitePendingDisconnect } = this.state
 
     disconnectAccount(sitePendingDisconnect.domainKey)
-    this.clearSitePendingDisconnect()
+    this.clearPendingDisconnect()
+  }
+
+  disconnectAllAccounts = () => {
+    const { disconnectAllAccounts } = this.props
+    const { sitePendingDisconnect } = this.state
+
+    disconnectAllAccounts(sitePendingDisconnect.domainKey)
+    this.clearPendingDisconnect()
   }
 
-  renderConnectedSites () {
+  renderConnectedSitesList () {
     return (
       <ConnectedSitesList
         connectedDomains={this.props.connectedDomains}
-        onDisconnectSite={this.setSitePendingDisconnect}
+        onDisconnect={this.setPendingDisconnect}
       />
     )
   }
 
-  render () {
-    const { accountLabel, history, legacyExposeAccount, tabToConnect, connectedDomains } = this.props
+  renderConnectedSitesPopover () {
+
+    const {
+      accountLabel,
+      closePopover,
+      connectedDomains,
+      legacyExposeAccount,
+      tabToConnect,
+    } = this.props
     const { t } = this.context
+
+    return (
+      <Popover
+        title={t('connectedSites')}
+        subtitle={connectedDomains.length
+          ? t('connectedSitesDescription', [accountLabel])
+          : t('connectedSitesEmptyDescription', [accountLabel])
+        }
+        onClose={closePopover}
+        footer={
+          tabToConnect
+            ? (
+              <div
+                className="connected-sites__text-button"
+                onClick={legacyExposeAccount}
+              >
+                {t('connectManually')}
+              </div>
+            )
+            : null
+        }
+        footerClassName="connected-sites__add-site-manually"
+      >
+        {this.renderConnectedSitesList()}
+      </Popover>
+    )
+  }
+
+  renderDisconnectPopover () {
+
+    const { closePopover, permittedAccountsByOrigin } = this.props
+    const { t } = this.context
+    const { sitePendingDisconnect: { domainKey, domainName } } = this.state
+
+    const numPermittedAccounts = permittedAccountsByOrigin[domainKey].length
+
+    return (
+      <Popover
+        className="connected-sites"
```

Comment:
We agreed that it does make sense, because it's the parent class name, and it was just missing from the other popover (all its children already used `connected-sites__`).

Reply:
`connected-sites` class added to other popover in b397aa5